src/onboarding2/Create/Create.js

```diff
@@ -441,6 +453,21 @@ function Create({ account, onOpenOrg, templates, walletWeb3, web3 }) {
           templates={templates}
         />
       )}
+      <ErrorModal
+        action={
+          <Button mode="strong" onClick={() => setAttempts(a => a + 1)}>
+            OK, let’s try again
+          </Button>
+        }
+        content={
+          <p>
+            An error has occurred during the signature process. Do not worry,
+            you can try again without losing your information.
```

Comment:
```suggestion
            you can try to send the transaction again.
```

cc @dizzypaty there's not so much "information" to show here